src/frontend/src/app/pagination/pagination.component.html

```diff
@@ -1,11 +1,11 @@
 <nav class="pagination is-centered" role="navigation" aria-label="pagination">
   <button class="button pagination-previous" (click)="onPrev()" [disabled]="page === 1">{{ page ==1 ? '': page-1 }}&nbsp;<i class="fa fa-caret-left" aria-hidden="true"></i></button>
-  <button class="button pagination-next" (click)="onNext()" [disabled]="lastPage()"><i class="fa fa-caret-right" aria-hidden="true">&nbsp; {{ page == totalPage ? '' : page + 1 }}</i></button>
+  <button class="button pagination-next" (click)="onNext()" [disabled]="lastPage()"><i class="fa fa-caret-right" aria-hidden="true"></i>&nbsp; {{ page == totalPage ? '' : page + 1 }}</button>  
```

Comment:
good catch!

Reply:
@monika-Dianomic We need to add class `is-mobile` in  `<div class="columns">`  every where we are using that.